libs/tokens/src/tokens/space/margins.json

```diff
@@ -0,0 +1,128 @@
+{
+    "sys": {
+        "space": {
+            "margins": {
+                "0": {
```

Comment:
Should these also be "xs, s, m, l....." maybe?

However they are many 😄 OK as is but lets think about it 👍

Reply:
I think the gap and paddings on grid it self make sense since is limited to certain amounts, but in here since the list is extensive I think is going to be difficult to name. But definitely we can look into it ✌️